### ISSUE TITLE
Analysis API artifacts have the wrong common-deps dependency

### ISSUE THREAD

**Issue by ZacSweers** [google/ksp#1712]
They both point at `com.google.devtools.ksp:common-deps` when they should point at `com.google.devtools.ksp:symbol-processor-common-deps` I think?

```
:ksp:main: Could not find com.google.devtools.ksp:common-deps:2.0.0-Beta3-1.0.17.
Searched in the following locations:
  - https://repo.maven.apache.org/maven2/com/google/devtools/ksp/common-deps/2.0.0-Beta3-1.0.17/common-deps-2.0.0-Beta3-1.0.17.pom
  - https://dl.google.com/dl/android/maven2/com/google/devtools/ksp/common-deps/2.0.0-Beta3-1.0.17/common-deps-2.0.0-Beta3-1.0.17.pom
Required by:
    project :ksp > com.google.devtools.ksp:symbol-processing-aa-embeddable:2.0.0-Beta3-1.0.17
```

**Comment by ZacSweers** [comment]
Here is a repro + workaround if it helps https://github.com/ZacSweers/kotlin-compile-testing/pull/196/

**Comment by ting-yuan** [comment]
Fixed in #1715
